Source/ACE/Entity/Door.cs

```diff
@@ -215,9 +215,6 @@ public override void OnUse(ObjectGuid playerId)
                 }
                 else
                 {
-                    bool isLocked;
-                    bool isOpen;
-
                     ActionChain checkDoorChain = new ActionChain();
                     CurrentLandblock.ChainOnObject(checkDoorChain, Guid, (WorldObject dwo) =>
```

Comment:
You should be able to do an action chain with "this", like you did with moveToDoorChain above, instead of using the CurrentLandblock.ChainOnObject (no need to bring the landblock into this when unneeded).

Otherwise this looks better now.